app/views/articles/_object.html.erb

```diff
@@ -1,6 +1,8 @@
 <div class="list-group-item">
-  <%= content_tag(:big, link_with_query(object.title.t,
-                  object.show_link_args)) %>
+  <%= content_tag(
+        :big,
+        link_to(object.title.t, article_path(object, q: get_query_param))
```

Comment:
I think that in `bootstrap-4` @nimmolo modified `link_with_query` to work with helper paths. 
I think he also made it work with `object_path` (as opposed to `article_path`). 
If so, should those be ported to `master`? But see my comments above above not being able to test ApplicationController.

Reply:
@JoeCohen I did add a module, in `models/concerns/routeable.rb` that makes the path helpers available anywhere when included. But I believe `object_link_helper` ended up not needing it. 

The module contains comments with links to articles explaining how to use it. (let's keep those comments for now!)